sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -1349,6 +1351,54 @@ int sys_fstatvfs(int fd, struct statvfs *out) {
 	return 0;
 }
 
+int sys_sysconf(int num, long *ret) {
+	switch(num) {
+		case _SC_OPEN_MAX: {
+			struct rlimit ru;
+			if(int e = sys_getrlimit(RLIMIT_NOFILE, &ru); e) {
+				return e;
+			}
+			*ret = (ru.rlim_cur == RLIM_INFINITY) ? -1 : ru.rlim_cur;
+			break;
+		}
+		case _SC_NPROCESSORS_ONLN: {
+			cpu_set_t set;
+			CPU_ZERO(&set);
+			if(int e = sys_getaffinity(0, sizeof(set), &set); e) {
```

Comment:
Why do we do it like this, though? This looks a lot like it could be in unistd-stubs, not in the Linux sysdeps.

Reply:
Not every sysdep might want to implement this, as this is a Linux-specific function.